docs/static/running-logstash-command-line.asciidoc

```diff
@@ -1,15 +1,40 @@
-[[command-line-flags]]
-=== Command-Line Flags
+[[running-logstash-command-line]]
+=== Running Logstash from the Command Line
 
-Logstash has the following flags. You can use the `--help` flag to display this information.
+To run Logstash from the command line, use the following command:
+
+[source,shell]
+----
+bin/logstash [options]
+----
+
+Where `options` are <<command-line-flags,command-line>> flags that you can
+specify to control Logstash execution. The location of the `bin` directory
+varies by platform. See <<dir-layout>> to find the location of `bin\logstash` on
+your system.
+
+The following example runs Logstash and loads the Logstash config defined in
+the `mypipeline.conf` file:
 
-Instead of specifying options at the command line, we recommend that you control Logstash execution
-by specifying options in the Logstash <<logstash-settings-file,settings file>>. Using a settings file
-makes it easier for you to specify mutliple options, and it provides you with a single, versionable
-file that you can use to start up Logstash consistently for each run.
+[source,shell]
+----
+bin/logstash -f mypipeline.conf
+----
+
+Specifying command line options is useful when you are testing Logstash.
```

Comment:
We should also say that any command line options specified will overwrite the configuration specified in the settings file. Also, if you provide a configuration via command line, the settings file is not changed and will remain as-is.